pkg/istio/virtualservicegatway_test.go

```diff
@@ -0,0 +1,125 @@
+package istio_test
+
+import (
+	"github.com/maistra/istio-workspace/pkg/apis/istio/v1alpha1"
+	"github.com/maistra/istio-workspace/pkg/istio"
+	"github.com/maistra/istio-workspace/pkg/log"
+	"github.com/maistra/istio-workspace/pkg/model"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	istionetworkv1alpha3 "istio.io/api/networking/v1alpha3"
+	istionetwork "istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("Location of Gateway connected VirtualService Kind", func() {
+
+	// Missing tests:
+	//  - verify existing host logic from GW works (filter out our own sub domains)
```

Comment:
Is that `// TODO` for this PR?

Reply:
https://github.com/maistra/istio-workspace/pull/469/commits/f006ece2d6ac587e091b04cd96c7c726fa1cecd3